Dockerfile

```diff
@@ -86,6 +86,12 @@ RUN sed -i 's/USE_HOME = False/USE_HOME = True/g' MobSF/settings.py && \
 
 # Postgres support is set to false by default
 ARG POSTGRES=False
+
+ENV POSTGRES_USER=postgres
```

Comment:
should the `ENV` fields be all removed?

Reply:
For me no, it provide default values if theres no en variable set by user